### DISCUSSION TITLE
Swiper is not defined

### DISCUSSION THREAD

**Discussion by IIamTyZen** [nolimits4web/swiper#5739] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/o0mxvt?file=/index.html

### Bug description

I have in the console this error:
Uncaught ReferenceError: Swiper is not defined
I don't know how to resolve this, if someone know how to do this, it's will be perfect,
Thanks You

### Swiper version

7

### Platform/Target and Browser Versions

Windows

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by Arhell** [comment]
just looked no errors
try
`main.js import './slider';`

slider.js:

```
import '../../node_modules/swiper/dist/js/swiper.min.js';

var mySwiper = new Swiper(document.querySelector('.swiper-container'), {
    slidesPerView: 1,
    speed: 500,
    navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
    },
    autoplay: {
        delay: 2000,
      },
});
```

**Comment by PatAbah** [comment]
Same issue. 

Just define your stuff directly in the dom with className "mySwiper". E.g
`<swiper-container class="mySwiper" navigation="true" pagination="true" css-mode=True etc>
           ... `

Might help 🙂
